owl_example/lib/json_example.dart

```diff
@@ -56,6 +56,17 @@ class ChildClass {
   ///
   @JsonField(native: true)
   Map map;
+
+  @Transient()
+  Map _virtualField;
+
+  ///
+  @JsonField(native: true)
+  Map get virtualNativeField => _virtualField;
+
+  ///
+  @JsonField(native: true)
```

Comment:
AFAICT this should work if only the getter has the annotation (there is no need to duplicate the same annotation, right)?

Reply:
Yes. I have tested that case locally, but was just being explicit in this test case.

Reply:
Could you remove the one on the setter, add a comment that "It is enough to annotate the getter."? I'll be happy to take it without an explicit test.